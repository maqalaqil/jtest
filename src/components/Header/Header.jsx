import React from 'react'

export default function Header() {
    return (
        <div>
            <h1>hello world</h1>
            <h2>this is for testing</h2>
            <h3>subscribe to my channel</h3>
        </div>
    )
}
