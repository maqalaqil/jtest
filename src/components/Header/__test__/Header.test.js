import * as React from "react";
import * as renderer from "react-test-renderer";
import Header from '../Header'

it("this is working head", () => {
  const tree = renderer.create(<Header />).toJSON();
  expect(tree).toMatchSnapshot();
});