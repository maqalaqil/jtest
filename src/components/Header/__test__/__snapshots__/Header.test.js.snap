// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`this is working head 1`] = `
<div>
  <h1>
    hello world
  </h1>
  <h2>
    this is for testing
  </h2>
  <h3>
    subscribe to my channel
  </h3>
</div>
`;
